scripts/jest/setupTests.www.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+jest.mock('shared/ReactFeatureFlags', () => {
+  jest.mock(
+    'ReactFeatureFlags',
+    () => jest.requireActual('shared/forks/ReactFeatureFlags.www-dynamic'),
+    {virtual: true}
+  );
+
+  const wwwFlags = jest.requireActual('shared/forks/ReactFeatureFlags.www');
+  const defaultFlags = jest.requireActual('shared/ReactFeatureFlags');
+
+  // TODO: Many tests were written before we started running them against the
+  // www configuration. Update those tests so that they work against the www
+  // configuration, too. Then remove these overrides.
+  wwwFlags.disableLegacyContext = defaultFlags.disableLegacyContext;
+  wwwFlags.warnAboutUnmockedScheduler = defaultFlags.warnAboutUnmockedScheduler;
+  wwwFlags.enableUserTimingAPI = defaultFlags.enableUserTimingAPI;
+  wwwFlags.disableJavaScriptURLs = defaultFlags.disableJavaScriptURLs;
+  wwwFlags.enableDeprecatedFlareAPI = defaultFlags.enableDeprecatedFlareAPI;
```

Comment:
@trueadm When I ran the tests with Flare enabled, this flag started failing sporadically:

```js
Summary of all failing tests
 FAIL  packages/react-dom/src/__tests__/ReactDOMFiber-test.js
  ● ReactDOMFiber › should not update event handlers until commit

    Expected warning was not recorded:
      "Warning: unstable_flushDiscreteUpdates: Cannot flush updates when React is already rendering."

      1051 |         expect(() => {
      1052 |           node.click();
    > 1053 |         }).toErrorDev(
           |            ^
      1054 |           'Warning: unstable_flushDiscreteUpdates: Cannot flush updates when React is already rendering.',
      1055 |         );
      1056 |       }
```

Details: https://circleci.com/gh/facebook/react/97022?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Not sure if this is an issue or not, but letting you know just in case. Since Flare is enabled in the www build.

Reply:
This is known, and it seemed like maybe a fragile test from memory as it’s related to the discrete flushing logic based on the timeStamp logic.